spec/serializers/degree_serializer/v01_spec.rb

```diff
@@ -9,7 +9,6 @@
   describe "serialization" do
     it "includes all expected fields" do
       %w[
-        degree_id
```

Comment:
I thought that we would always get a `degree_id` with a Degree. Do you know what circumstances we don't get this?

Reply:
I saw that the ID was in the list of excluded attributes, and the same for placements, so I took them out. Shall I put them both back? It's just the ID from the database right?

Reply:
Ah, I think I can see the confusion. You're right that we don't want the degree `id` but we **do** want the `degree_id` which internally is the `slug`.